hws/src/dr_common.py

```diff
@@ -152,10 +152,11 @@ def hex_to_bin_str(_n, _len):
 STC_ACTION_JUMP_TO_VPORT = '85'
 
 
-def hit_addr_calc(next_table_base_39_32, next_table_base_31_5):
-    hit_addr = next_table_base_39_32 << 32
+def hit_addr_calc(next_table_base_63_48, next_table_base_39_32, next_table_base_31_5):
+    hit_addr = next_table_base_63_48 << 40
```

Comment:
why this is << 40 and not 48

Reply:
and why you need the GVMI in the address, which use case?

Reply:
> why this is << 40 and not 48

My understanding is: The bits are physically located at bit 48 of the STE layout, simply because the address has some other field interspersed in the middle of it. However, if you count the bits "below" this field, there are 40 of them not 48.

Perhaps bits 47..40 of the address are not represented in the STE, and cannot be set, and hence are always 0, but I don't _think_ that's true.

> and why you need the GVMI in the address, which use case?

The destination GVMI can vary. In order to interpret what the "program" actually does, one must understand exactly what is actually being jumped to, not make invalid assumptions that the GVMI never changes. For example:
* FW-defined STEs that implement certain pre-defined actions (e.g. DROP) are in GVMI 0x8111 not the "current" GVMI.
* I believe inter-domain jumps such as "go to port/VF" are cross-GVMI.
* Other reasons I can explain in a non-public context.

Reply:
The latest code doesn't combine the GVMI and index, so this comment no longer applies; closing.